service.c

```diff
@@ -283,7 +283,13 @@ CheckIServiceStatus(BOOL warn)
     {
         /* warn that iservice is not started */
         if (warn)
-            ShowLocalizedMsg(IDS_ERR_NOTSTARTED_ISERVICE);
+        {
+            if (IsUserAdmin())
```

Comment:
We have to include misc.h to use this function here.

Reply:
Added #include.